src/pages/Register.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import ChevronRightOutlinedIcon from '@mui/icons-material/ChevronRightOutlined';
+import { useApi } from '../hooks/useApi';
+import { useGlobalContext } from '../context';
+import { useNavigate } from 'react-router-dom';
+
+const validate = (values) => {
+  const errors = {};
+  if (!values.password) {
+    errors.password = 'Password Required';
+  } else if (values.password.length < 6) {
+    errors.password = 'Must be 6 characters or more';
+  }
+
+  if (!values.email) {
+    errors.email = 'E-mail Required';
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = 'Invalid email address';
+  }
+
+  if (values.password !== values.password2) {
+    errors.password2 = 'Passwords must match';
+  }
+
+  return errors;
+};
+
+const Register = () => {
+  const navigate = useNavigate();
+  const request = useApi();
+  const { userInfo } = useGlobalContext();
+  console.log(userInfo);
```

Comment:
If you remove this console.log, the `useGlobalContext()` hook is no longer needed, it's not used anywhere in the file.